vege/style.css

```diff
@@ -34,6 +35,157 @@ body {
 }
 
 
+
+
 /* ------------------------------- */
 /* ----- DOLE BUDE TVOJE CSS ----- */
 /* ------------------------------- */
+.container {
+    flex-direction: column;
+}
+h1 {
+    font-weight: 800;
+    font-size: 1.75rem;
+    padding: 10rem 0.5rem 0.5rem 0.5rem;
+    margin: 0;
+}
+h2 {
+    font-weight: 800;
+    font-size: 1.2rem;
+    padding: 5rem 0.5rem 0.5rem 0.5rem;
+    margin: 0;
+}
+p1 {
```

Comment:
Paragraf (odstavec) se typicky nečísluje - prostě jen <p> ale očividně to funguje i s tím. Nezaručuju ale, že by s tím jinde nebyly problémy... :)

Reply:
Tady mě to očíslování paragrafu napadlo, protože jedno mám float: left;  a druhé float: right; říkala jsem si, že by to možná mohlo být přímočařejší než si vytvářet a pojmenovávat třídy a mohlo by to fungovat. Byl to takový momentální nápad abych s každým tím textem mohla pracovat trošku jinak.